airflow/settings.py

```diff
@@ -341,11 +341,11 @@ def configure_action_logging():
     :rtype: None
     """
 
-try:
-  from airflow_local_settings import *
-  log.info("Loaded airflow_local_settings.")
-except Exception as e:
-  log.error('Failed to load airflow_local_settings because of: %s', str(e))
+# try:
```

Comment:
Should this be commented out?

Reply:
I think we use this in airflow, @smith-m's change I believe relies on this.

Reply:
I commented it out because I was getting an exception for not having airflow local settings defined

Reply:
This might be how Airflow policies are loaded - both the the sensor configuration override and event_airflow_task_lifecycle rely on this and are a part of Airflow capabilities.

It looks like this exception is caught - does it need to be commented out?

Reply:
I'll revert my change here then. Thanks for flagging this